src/libraries/System.Private.CoreLib/src/System/Threading/PreAllocatedOverlapped.Browser.cs

```diff
@@ -7,8 +7,13 @@ public sealed class PreAllocatedOverlapped : System.IDisposable
     {
         [CLSCompliantAttribute(false)]
         public PreAllocatedOverlapped(IOCompletionCallback callback, object? state, object? pinData) { }
+#pragma warning disable CA1822 // Mark members as static
+        internal bool AddRef() => false;
+        internal void Release() { }
+#pragma warning restore CA1822 // Mark members as static
         [CLSCompliantAttribute(false)]
         public static PreAllocatedOverlapped UnsafeCreate(IOCompletionCallback callback, object? state, object? pinData) => new PreAllocatedOverlapped(callback, state, pinData);
         public void Dispose() { }
+        internal ThreadPoolBoundHandleOverlapped? _overlappedPortableCore;
```

Comment:
Should more of the Mono Browser thread pool be moved too to make these changes unnecessary?